docs/internal-documentation/maintenance/SystemTesting.md

```diff
@@ -45,17 +45,17 @@ The set of configurations must include at least:
 ### Procedure
 
 1. Make sure you have the correct preliminary distribution tarball for each
-   component.
+   component. See [Create Test Distribution].
 
 2. For each configuration:
 
    1. Prepare a machine
       1. Allocate a machine with the architecture specified by the configuration.
       2. Follow the preparation document for the operating system specified by the configuration.
-         * CentOS-Preparation.md (unavailable at this time)
-         * [Debian-Preparation.md](https://github.com/zonemaster/zonemaster/blob/master/docs/internal-documentation/distrib-testing/Debian-Preparation.md)
-         * [FreeBSD-Preparation.md](https://github.com/zonemaster/zonemaster/blob/master/docs/internal-documentation/distrib-testing/FreeBSD-Preparation.md)
-         * Ubuntu-Preparation.md (unavailable at this time)
+         * [CentOS-Preparation]
+         * [Debian-Preparation]
+         * [FreeBSD-Preparation]
+         * [Ubuntu-Preparation]
```

Comment:
Rocky Linux is missing here. If we do not have a document, then it would be reasonable to add
```
* Rocky-Linux-Preparation (TBD)
```